TTPChallenge/QuizScreen.storyboard

```diff
@@ -0,0 +1,1995 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
The quiz should not be a modal, it should be a "push." It is odd for the start button to have a > arrow and then for the quiz to come up from the bottom. Modals are typically used for screens that do not navigate to anything else. Since this screen continues on, it should be a push.